.github/workflows/ci.yml

```diff
@@ -148,3 +155,44 @@ jobs:
 
           gsutil cp LATEST.tmp gs://$BUCKET/$BUCKET_PATH/RELEASED
           gsutil cp LATEST.tmp gs://$BUCKET/$BUCKET_PATH/RELEASED.$BUILD_VERSION
+
+  deploy:
+    if: |
+      github.event.inputs.deploy == 'true' ||
+      github.event_name == 'release' ||
+      github.event_name == 'push' && github.ref_name == 'master' ||
+      github.event_name == 'push' && startsWith(github.ref_name, 'release-v')
+
+    # Only deploy if all tests passed.
+    needs: [ci]
+
+    runs-on: ubuntu-latest
+
+    steps:
+      # No need to checkout the project, since all we need is to download the
+      # build artifact from the build step.
+      - uses: actions/download-artifact@v3
+        with:
+          name: build
+          path: build
+
+      - name: Disable Jekyll
+        run: |
+          touch build/.nojekyll
+
+      - name: Untar
+        run: |
+          cd build
+          tar x -zf $BUILD_VERSION.$ARTIFACT_EXTENSION
+          rm $BUILD_VERSION.$ARTIFACT_EXTENSION
+
+      # This seems to be the simplest way to publish to a separate branch.
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_branch: gh-pages
+          publish_dir: ./build
+          # Optional. This will create a CNAME file so GitHub Pages serves it
```

Comment:
Drop this?

Reply:
Changed it to docs.toit.io, but commented it out.